test/SampleAggregatorTest.cpp

```diff
@@ -365,6 +365,90 @@ TEST_F(SampleAggregatorTest, epoch_application_total)
     EXPECT_DOUBLE_EQ(7.0, m_agg->sample_application(M_SIGNAL_TIME));
 }
 
+TEST_F(SampleAggregatorTest, invalid_value_total)
```

Comment:
Does this need to use a domain like package to reproduce the issue we are seeing in the integration test?  The Failure in the integration test is specifically about signals at the socket domain.

Reply:
@cmcantalupo We'll discuss it during the meeting.